tests/legacy-cli/e2e/ng-snapshot/package.json

```
@@ -11,7 +11,7 @@
     "@angular/forms": "github:angular/forms-builds#ba125154b807fc8256258e7ff2407c3d1e633ce5",
     "@angular/language-service": "github:angular/language-service-builds#dbc2805aa0175bc51854b65596d95dea9005a5b4",
     "@angular/localize": "github:angular/localize-builds#5ac5d144ee11b00d5980910dcab4e041c99f5dad",
-    "@angular/material": "github:angular/material2-builds#cede9300b8ff2e7b3d728e49a28084638d9a7533",
+    "@angular/material": "github:angular/material-builds#cede9300b8ff2e7b3d728e49a28084638d9a7533",
```

```suggestion
    "@angular/material": "github:angular/material-builds#4e922846f794a959b9d6e33d42cf69edbf097f14",
```
Latest SHA for https://github.com/angular/material-experimental-builds/commits/13.2.x

---

won't renovate do that?

---

Renovate only updates the main branch but we can leave it as-is based on the scope of this PR.